numba_cuda/numba/cuda/tests/test_binary_generation/generate_raw_ltoir.py

```diff
@@ -160,7 +163,9 @@ def main(sourcepath, outputpath, arch):
         help="compute arch to target (e.g. sm_87). Defaults to sm_50.",
         default="sm_50",
     )
-
+    parser.add_argument(
+        "--nrt", type=lambda x: x.lower() == "true", default=False
+    )
```

Comment:
Could this be:
```suggestion
    parser.add_argument("--nrt", action="store_true")
```

to keep it simple? I'm not sure I understand what is going on with the lambda here.

Reply:
Just some leftover string validation from a previous iteration here I believe.